documentation/source/user_section/tutorials/registration-to-template/lumbar-registration/labeling.rst

```diff
@@ -49,6 +49,13 @@ For this image, the coordinate ``[27,79,80]`` seems to be appropriate location:
 .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lumbar-registration/label-selection-conus-medullaris.png
    :align: center
 
+..
+   comment:: The script and course slides mention the coordinate ``[27,76,187]``,
+             but the figure below explicitly uses the coordinate ``[22,77,187]``.
+             Given that updating the tutorial to match the script/slides would
+             require updating the figure, I decided to leave it for now and
+             double-check why the slides were changed in the first place.
```

Comment:
re: 686503935d30b32bbd5518f3a90ec270cab8d394; I feel like I didn't explore the corresponding `sct_tutorial_data` commit close enough (https://github.com/spinalcordtoolbox/sct_tutorial_data/commit/3ecf3147cc7521389cb9b536ac79899470811c73). Why did we change this again? :thinking: